.gitignore

```diff
@@ -63,3 +63,8 @@ Assets/ThirdParty.meta
 Assets/TextMesh Pro.meta
 Assets/TextMesh Pro/
 --Version/
+
+# ================= #
+# Android generated #
+# ================= #
+*.apk
```

Comment:
Shouldn't this have been left out if one were to build to a build(s) folder?

Reply:
I am somewhat new to building Android applications with unity. I will say that file explorers default path when launched from build settings directs to the repo root, which can cause apks to accidentally get placed out of what may be the best practice directory. Are there any expected scenarios for checking in apps?

Reply:
It's really the devs choice. In the build window it is pretty smart about putting it in the build/builds folder so it's automatically excluded from the repository.

This change is not needed

Reply:
I've gone ahead and removed this change. We will have to be diligent about not having accidental apk's get checked in by contributors. But I don't have a strong preference and can see how this change was more for my own convenience.